Agile Development/Burndown Chart/script.py

```diff
@@ -0,0 +1,132 @@
+import pprint
+import requests
+import jpholiday
+import datetime
+import matplotlib.dates as mdates
+import pandas as pd
+import matplotlib.pyplot as plt
+import urllib.request
+import urllib.parse
+import json
+import datetime
```

Comment:
you're already importing datetime on line 4

Reply:
Fixing duplicate imports of datetime